File: app/src/main/java/com/scatl/uestcbbs/module/user/view/ModifyAvatarView.java
package com.scatl.uestcbbs.module.user.view;

import com.scwang.smart.refresh.layout.SmartRefreshLayout;

public interface ModifyAvatarView {
    void onGetParaSuccess(String agent, String input);
    void onGetParaError(String msg);
    void onUploadSuccess(String msg);
    void onUploadError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/user/view/ModifyAvatarView.kt
package com.scatl.uestcbbs.module.user.view

import com.scatl.uestcbbs.base.BaseView

/**
 * Created by sca_tl at 2023/6/14 17:05
 */
interface ModifyAvatarView: BaseView {
    fun onGetParamsSuccess(agent: String?, input: String?)
    fun onGetParamsError(msg: String?)
    fun onUploadSuccess(msg: String?)
    fun onUploadError(msg: String?)
}